src/modules/core/components/link/link.tsx

```diff
@@ -0,0 +1,12 @@
+import type { FC } from 'react';
+
+import type { LinkProps } from './link.interface';
+import styles from './link.module.scss';
+
+export const Link: FC<LinkProps> = ({ href, children }) => {
```

Comment:
Do we really need this component? Next already have link https://nextjs.org/docs/pages/api-reference/components/link